include/SimpleTextFloat.h

```diff
@@ -31,7 +31,7 @@
 #include "lmms_export.h"
 
 class QLabel;
-class Qtimer;
+class QTimer;
```

Comment:
The fact that it did compile with the typo `Qtimer` shows that the forward declaration is technically not needed as the class is already forward declared or included via one of the other includes.

I think it's still cleaner to keep the forward include in this header as well in case it is removed from one of the other headers. Don't know what's the actual code policy though.